site/utils/formUtils.ts

```diff
@@ -84,3 +85,27 @@ export const getStopType = (stopType: string | undefined) => {
         return "Stop";
     }
 };
+
+// eslint-disable-next-line @typescript-eslint/require-await
+export const filterServices = async (servicesData?: Service[]) => {
```

Comment:
can you remove the async

Reply:
If async was removed then the method would run synchronously and not in parallel

Reply:
I don't think it will make too much of a difference to performance in this case